src/mrack/run.py

```diff
@@ -196,6 +197,29 @@ async def ssh(ctx, hostname, metadata):
     ssh_action.ssh(hostname)
 
 
+@mrackcli.group()
+async def eh():
```

Comment:
The naming could be better, can we use `hosts` for example ?

Reply:
I considered "hosts" but find it misleading as mrack's main functionality is about provisioning hosts and thus people might expect to work with the hosts somehow.  Then something like `etc-hosts` is too long and difficult to type. Thus ended with `eh` which is easy to remember and fast to type.

Reply:
What about `patch-hosts` i do not mind at all that it is little bit longer when it says the purpose.

Reply:
`patch-hosts` is the same as `etc-hosts`, maybe actually worse as is longer. What could be OK, is to have a longer alias, i.e. both `eh` and `etc-hosts`. But really, don̈́t see a need, current help is quite self-explanatory.

```
$ mrack
Usage: mrack [OPTIONS] COMMAND [ARGS]...

  Multihost human friendly provisioner.

Options:
  -c, --mrack-config PATH
  -p, --provisioning-config PATH
  -d, --db PATH
  --debug
  --help                          Show this message and exit.

Commands:
  destroy  Destroy provisioned hosts.
  eh       Commands to update /etc/hosts file.
  list     List host tracked by.
  output   Create outputs - such as Ansible inventory.
  ssh      SSH to host.
  up       Provision hosts.


$ mrack eh
Usage: mrack eh [OPTIONS] COMMAND [ARGS]...

  Commands to update /etc/hosts file.

Options:
  --help  Show this message and exit.

Commands:
  add    Add active hosts to /etc/hosts file.
  clear  Remove all mrack hosts from /etc/hosts file.
```